docker/build/Dockerfile

```diff
@@ -19,6 +16,8 @@ COPY --from=golangci/golangci-lint:v1.23.7 /usr/bin/golangci-lint /usr/local/bin
 RUN wget -O /usr/local/bin/kind https://github.com/kubernetes-sigs/kind/releases/download/v0.11.1/kind-linux-amd64 \
     && chmod +x /usr/local/bin/kind
 
+RUN apt-get clean
```

Comment:
Why not include the `apt-get clean` invocation in the `RUN` statement above?
This may be (a) adding an extra layer; and (b) making larger-than-needed the layer for the first `RUN` statement above.

Reply:
You are right